spec/game_runner_spec.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'spec_helper.rb'
+require_relative '../lib/game_runner.rb'
+
+RSpec.describe Game, "#run" do
+  def grid_format
+    (<<~GRID)
+   __ __ __
+  |  |  |  |
+  |__|__|__|
+  |  |  |  |
+  |__|__|__|
+  |  |  |  |
+  |__|__|__|
+    GRID
+  end
+  context "on start up" do
+    it "renders an empty grid on the screen" do
+      game = Game.new
+
+      game.run
+
+      expect(game.run).to eq(grid_format)
```

Comment:
You are calling `game.run` twice which is confusing, and I think accidental. Also, I am not expecting running the game to return anything. It is going to interact with the command line. 

Look at the README around testing. You need to inject stdin and stdout in order to be able to test that it is doing the right thing. 

These tests won't produce something that can be called and print a grid to the command line. It will just return a string.

Reply:
✅